canlink/handler.go

```diff
@@ -0,0 +1,6 @@
+package canlink
+
+type Handler interface {
+	Name() string
```

Comment:
What's the Name for?

Reply:
These are useful in logging, this way you can see the underlying handler implentation in a log line.